KinSDK/KinSDK/source/blockchain/StellarEventSource.swift

```diff
@@ -237,7 +237,7 @@ public final class StellarEventSource: NSObject, URLSessionDataDelegate {
             }
         }
         else {
-            emitter.finish()
+            emitter?.finish()
```

Comment:
This fix is already pending at https://github.com/kinecosystem/StellarKit/pull/22
:)